redwood-treehouse/src/jvmMain/kotlin/app/cash/redwood/treehouse/FrameClockServiceJvm.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.treehouse
+
+// TODO(jwilson): deduplicate with identical symbol in hostMain
```

Comment:
Is something preventing this from working as-is?

Reply:
That's a zipline source set for sharing between Android and JVM. We don't need that. Seems like JVM is missing from the host source set. I didn't even realize we had JVM enabled for Treehouse. I guess it comes by accident.

Reply:
Might be useful for testing? I’m not using it.